exporter/elasticsearchexporter/internal/serializer/otelserializer/otelserializer.go

```diff
@@ -3,9 +3,65 @@
 
 package otelserializer // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/elasticsearchexporter/internal/serializer/otelserializer"
 
-type Serializer struct{}
+import (
+	"fmt"
+	"time"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/exporter/elasticsearchexporter/internal/lru"
+)
+
+const (
+	knownExecutablesCacheSize             = 128 * 1024
```

Comment:
```suggestion
	knownExecutablesCacheSize             = 16 * 1024
```

Reply:
We see significantly less executables than frames and traces, apart from corner cases.